### PR TITLE
(PUP-10897) Update forcelocal documentation

### PR DESCRIPTION
**Pull request by ciprianbadescu**

### CODE REVIEW

**Review by joshcooper — APPROVED** [review]
LGTM, I'll merge tomorrow unless @clairecadman has updates

File: lib/puppet/provider/group/groupadd.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@
 require 'puppet/util/libuser'
 
 Puppet::Type.type(:group).provide :groupadd, :parent => Puppet::Provider::NameService::ObjectAdd do
-  desc "Group management via `groupadd` and its ilk. The default for most platforms."
+  desc "Group management via `groupadd` and its ilk. The default for most platforms.
+
+  In order to use `forcelocal` parameter, the `libuser` package (providing
```
Comment (clairecadman) [inline lib/puppet/provider/group/groupadd.rb:7]:
```suggestion
 To use the `forcelocal` parameter, you need to install the `libuser` package (providing
```

File: lib/puppet/provider/group/groupadd.rb [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+  `/usr/sbin/lgroupadd` and `/usr/sbin/luseradd`) must be installed."
```
Comment (clairecadman) [inline lib/puppet/provider/group/groupadd.rb:8]:
```suggestion
  `/usr/sbin/lgroupadd` and `/usr/sbin/luseradd`)."
```

File: lib/puppet/provider/user/useradd.rb [hunk 3]
```diff
@@ -7,7 +7,10 @@
 Puppet::Type.type(:user).provide :useradd, :parent => Puppet::Provider::NameService::ObjectAdd do
   desc "User management via `useradd` and its ilk.  Note that you will need to
     install Ruby's shadow password library (often known as `ruby-libshadow`)
-    if you wish to manage user passwords."
+    if you wish to manage user passwords.
+
+    In order to use `forcelocal` parameter, the `libuser` package (providing
```
Comment (clairecadman) [inline lib/puppet/provider/user/useradd.rb:12]:
```suggestion
    To use `forcelocal` parameter, you need to install the `libuser` package (providing
```

File: lib/puppet/provider/user/useradd.rb [hunk 4]
```diff
@@ -7,7 +7,10 @@
...
+    `/usr/sbin/lgroupadd` and `/usr/sbin/luseradd`) must be installed."
```
Comment (clairecadman) [inline lib/puppet/provider/user/useradd.rb:13]:
```suggestion
    `/usr/sbin/lgroupadd` and `/usr/sbin/luseradd`)."
```

**Comment by clairecadman** [comment]
@ciprianbadescu I made a couple of suggestions, but otherwise looks good!

File: lib/puppet/provider/user/useradd.rb [hunk 5]
```diff
@@ -7,7 +7,10 @@
 Puppet::Type.type(:user).provide :useradd, :parent => Puppet::Provider::NameService::ObjectAdd do
   desc "User management via `useradd` and its ilk.  Note that you will need to
     install Ruby's shadow password library (often known as `ruby-libshadow`)
-    if you wish to manage user passwords."
+    if you wish to manage user passwords.
+
+    To use `forcelocal` parameter, you need to install the `libuser` package (providing
```
Comment (GabrielNagy) [inline lib/puppet/provider/user/useradd.rb:12]: to match the other docstring
```suggestion
    To use the `forcelocal` parameter, you need to install the `libuser` package (providing
```

**Review by GabrielNagy — APPROVED** [review]

**Review by clairecadman — APPROVED** [review]

**Pull request merged**